content/pages/work-with-us.html

```diff
@@ -118,27 +176,20 @@ <h3 class="text-normal"> Feel empowered to continue with our guides</h3>
 
 <section class="usa-section bg-base-lightest">
   <div class="grid-container">
-    <div class="grid-row">
-      <div class="tablet-lg:grid-col-7">
-         {{ tech-adapt | markdownify }}
-      </div>
-      <div class="tablet-lg:grid-col-5">
-        {% image "assets/img/work-with-us/work-with-us-illo-3.svg" "" %}
-      </div>
-    </div>
     <h3 class="text-normal">Read related case studies</h3>
     <p class="font-sans-lg">Read short project summaries demonstrating how we’ve helped some of the largest federal agencies.</p>
     <div class="grid-row grid-gap-md">
     {% assign projects = 'fec-gov, treasury-data-act' | split: ", " %}
     {% for project in projects %}
       {% assign project_details = collections.services | where: 'fileSlug', project | first %}
       {% assign project_agency = project_details.data.agencies | where: "name", project_details.data.agency | first %}
+      {% assign project_permalink = project_details.data.permalink | url %}
       <div class="grid-col-12 tablet:grid-col-6 margin-top-3 tablet:margin-top-0">
         {% include "card-with-image.html",
            image_path: project_agency.logo,
            image_alt_text: project_agency.name,
            image_size: "md",
-           link_url: project_details.data.permalink,
+           link_url: project_permalink,
```

Comment:
clarification: What motivated the change here?

Reply:
The links were broken on the feature branch when the preview page was built, because of the absolute URLs that weren't filtered. Caley helped me troubleshoot that. I figured it would be a good idea if someone was new to editing this page later and wondered what broke if it works in prod.

Reply:
Thank you, that clarification confirms my original suspicion, which is, this is not the right place to make this change. Instead of typing out a bunch of explanation, I'll just issue a PR to refine this. Thanks for catching this in the first place, though.